assets/js/blocks/price-filter/edit.js

```diff
@@ -24,9 +25,15 @@ import { IconMoney, IconExternal } from '../../components/icons';
 import ToggleButtonControl from '../../components/toggle-button-control';
 
 export default function( { attributes, setAttributes } ) {
-	const getInspectorControls = () => {
-		const { showInputFields, showFilterButton } = attributes;
+	const {
+		className,
+		heading,
+		headingLevel,
+		showInputFields,
+		showFilterButton,
+	} = attributes;
```

Comment:
There's a one other place in this component that you missed that is using the destructured variable (in `getInspectorControls`).

Reply:
Fixed in 88e4a3bc7aacd5c1ace89e1b9cf30474a145e30e as well.